analysis/data_utils.py

```diff
@@ -26,8 +26,15 @@ def validate_data(experiment_df):
         raise EmptyDataError('Empty experiment data.')
 
     expected_columns = {
-        'experiment', 'benchmark', 'fuzzer', 'trial_id', 'time_started',
-        'time_ended', 'time', 'edges_covered'
+        'experiment',
+        'benchmark',
+        'fuzzer',
+        'trial_id',
+        'time_started',
+        'time_ended',
+        'time',
+        'edges_covered',
+        'crash_key',
```

Comment:
I'm somewhat surprised this isn't multiple keys, how can this be?

Reply:
This is a different attribute crash_key which we join with Crash table, see analysis/queries.py. We can't use crashes since it is just a relationship, also makes it hard to work as a dataframe.